Pages/Language.cs

```diff
@@ -1,171 +1,329 @@
-﻿using OpenQA.Selenium;
+﻿using Marsproject.Hooks;
+using Marsproject.Support;
+using OpenQA.Selenium;
 using OpenQA.Selenium.Support.UI;
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Security.Cryptography;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
+
 
 namespace Marsproject.Pages
 {
-    public class Language
-
-    { 
-        //locators
-
-        public By Addnew = By.XPath("//*[contains(text(),'Add New')]");
-      
-        public By language = By.Name("name");
-       
-        public By level = By.Name("level");
-       
-        public By basic = By.XPath("// *[@value='Basic']");
-       
-        public By coversational = By.XPath("// *[@value='Conversational']");
+    public class Language : NewCommondriver
 
-        public By fluent = By.XPath("// *[@value='Fluent']");
-
-        public By native = By.XPath("// *[@value='Native/Bilingual']");
+    {
+        public Language(IWebDriver driver)
+        {
+            this.driver = driver;
 
-        public By add = By.XPath("// *[@value='Add']");
-        
-        public By cancel = By.XPath("// *[@value='Cancel']");
-       
-        public By update = By.XPath("//*[@value='Update']");
+        }
 
-        public By Delete= By.XPath("//div[@data-tab='first']//tbody[1]/tr/td[3]/span[2]/i");
+        public void cleanup()
+        {
 
-        public By Edit = By.XPath("//div[@data-tab='first']//tbody[last()]/tr/td[3]/span[1]/i");
+            try
+            {
+                int num = driver.FindElements(By.XPath("//*[@class=\"remove icon\"]")).Count();
+                for (int i = 0; i < num; i++)
+                {
 
-        public By Edit_data= By.XPath("//*[contains(text(),'German')]");
+                    IWebElement delete = driver.FindElement(By.XPath("//*[@class=\"remove icon\"]"));
+                    delete.Click();
 
 
+                }
+            }
+            catch (StaleElementReferenceException e)
+            {
+                Console.WriteLine("No elememt to delete");
 
+            }
+            catch (ElementNotInteractableException e)
+            {
+                Console.WriteLine("No data to delete");
+            }
 
 
+        }
         //First language
-        public void Firstlanguagemethod(IWebDriver driver, string language1)
+        public void Firstlanguagemethod(string language1)
```

Comment:
what is the difference between Firstlanguagemethod vs second and third, can you please rename them as they are doing different things!